### ISSUE TITLE
Frontend for SWB authentication

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/spectral-workbench#448]
As finalised in https://github.com/publiclab/mapknitter/issues/385, @IshaGupta18 is implementing the front end as 
![image](https://user-images.githubusercontent.com/20972099/56476727-4244a380-64ba-11e9-8163-a8f1b348b4e1.png)
for MK. We can use the same partial files here at SWB as the code should be exactly the same. 
Thanks all

**Comment by SidharthBansal** [comment]
We will come to this once https://github.com/publiclab/mapknitter/issues/385 is complete @IshaGupta18 
I hope this works for you. I think we might be able to create a first timer issue for this one. Do you agree with me?

**Comment by SidharthBansal** [comment]
I figured out there are some changes in the bootstrap layouts in SWB and MK so this issue is not ideal for fto eventhough code is similar. So, it has to be done by someone.

**Comment by IshaGupta18** [comment]
Yes @SidharthBansal once the entire login system is complete in MK (with all PRs merged), I would split it up in small FTOs!

**Comment by SidharthBansal** [comment]
Isha, MK PR is completely separate from the SWB PR.
Thanks
Sidharth

**Comment by IshaGupta18** [comment]
@SidharthBansal #455 is my PR for this, kindly take a look at it. Thanks a lot!
